message.go

```diff
@@ -101,6 +101,10 @@ type Message struct {
 	// encryption details).
 	Footer Annotations
 
+	// TransactionID can be used by senders to ensur a message is included as part of a transaction, started with
+	// [TransactionController.Declare].
+	TransactionID any
```

Comment:
Document any constraints on the acceptable types.

Reply:
Documented using the type system :)